src/Appwrite/Resque/Worker.php

```diff
@@ -165,22 +172,24 @@ public static function error(callable $callback): void
     protected function getProjectDB(Document $project): Database
     {
         global $register;
-        $database = $project->getAttribute('database', '');
-        $internalId = $project->getInternalId();
-        if (empty($database)) {
-            throw new \Exception('Database name not provided - cannot get database');
+
+        $pools = $register->get('pools'); /** @var \Utopia\Pools\Group $pools */
+
+        if ($project->isEmpty() || $project->getId() === 'console') {
+            return $this->getConsoleDB();
         }
 
-        $cache = $register->get('cache');
-        $dbPool = $register->get('dbPool');
-        $namespace = "_$internalId";
-        $pdo = $dbPool->getPDO($database);
-        $dbForProject = DatabasePool::wait(
-            DatabasePool::getDatabase($pdo, $cache, $namespace),
-            'projects'
-        );
```

Comment:
Do we have the wait functionality written somewhere else ? We have to wait for the connection to be ready else it throws an exception

Reply:
We now create all the connections on server startup